symengine/polys/uintpoly_flint.cpp

```diff
@@ -39,18 +38,18 @@ int UIntPolyFlint::compare(const Basic &o) const
     return 0;
 }
 
+static const fz_t zero_poly(0);
```

Comment:
Yes, but it is being used in line 48, to return a zero polynomial (`UIntPolyFlint`). Also, being used in the `get_coeff_ref` method, so I feel that it should stay as a static member inside that function itself (just like before).